src/Compilers/CSharp/Portable/Emitter/Model/PEAssemblyBuilder.cs

```diff
@@ -15,6 +18,7 @@ internal abstract class PEAssemblyBuilderBase : PEModuleBuilder, Cci.IAssemblyRe
     {
         private readonly SourceAssemblySymbol _sourceAssembly;
         private readonly ImmutableArray<NamedTypeSymbol> _additionalTypes;
+        private ImmutableDictionary<WellKnownMember, MethodSymbol> _embeddedAttributesConstructors;
```

Comment:
Per discussion, the guidance is to not use a dictionary to store the attribute for now, but properties, and the design can be changed later if needed when more attributes are introduced.

---
In reply to: [113818380](https://github.com/dotnet/roslyn/pull/18715#discussion_r113818380) [](ancestors = 113818380,113572450)